src/erudio-client.ts

```diff
@@ -75,6 +78,14 @@ export class ErudioClient {
         structure.contentId,
         locale,
       );
+      if (
+        structure.contentType === 'learning-path' &&
+        structureContents.content.learningPath?.type === 'ref'
+      ) {
+        learningPath = await new LearningPath(this.host).getlearningPath(
```

Comment:
@nataszczypiora it shouldn't break if laerningPath does not exist or the service is down, you should catch and log errors

Reply:
try catch added